charts/karmada/templates/karmada-search.yaml

```diff
@@ -87,6 +90,14 @@ spec:
           resources:
           {{- toYaml .Values.apiServer.resources | nindent 12 }}
       volumes:
+        - name: etcd-cert
+          secret:
+          {{- if eq .Values.etcd.mode "internal" }}
+            secretName: {{ $name }}-cert
+          {{- end }}
+          {{- if eq .Values.etcd.mode "external" }}
```

Comment:
If the names are neither `internal` nor `external`, It would be broken.

Got the error yaml:

```yaml
...
        - name: etcd-cert
          secret:
...
```

Reply:
Yes, but in order for search to access etcd, you have to configure an etcd mode, and there is a problem if you don't configure the service, there is certificate related in the etcd configuration, but I think the same configuration, since it already exists, can be reused or is there a better way to handle it

Reply:
I have check the repo and it's really exist, so the PR can keep it.

https://github.com/karmada-io/karmada/blob/5f96d733ba4b3ed80d68c2e1ade8ecfe2f1c9e77/charts/karmada/templates/karmada-aggregated-apiserver.yaml#L103-L110